src/index.js

```diff
@@ -47,7 +49,9 @@ function App() {
 
   return (
     <Security {...config} onAuthRequired={authHandler}>
-      <Navigation />
+      {/* hides the Navigation menu when at the login page, since the login page's purpose is singular */}
+      {location.pathname === '/login' ? null : <Navigation />}
```

Comment:
Good catch in fixing this. Now it seems like the Navbar won't be loading on the login page, it was a really weird bug. 

However, since we are converting pages to And D layout, we can be more deliberate in our placement of `<Navigation/>` and removing this bug as a byproduct. Do you consider merging or closing the pull request @stkirk @ashtilawat23 ?